### ISSUE TITLE
INPC005 should not warn here

### ISSUE THREAD

**Issue by JohanLarsson** [DotNetAnalyzers/PropertyChangedAnalyzers#36]
```cs
        public double EncoderPositionY
        {
            get => _encoderPositionY;

            set
            {
                if (Math.Abs(value - _encoderPositionY) < 1e-6)
                {
                    return;
                }

                _encoderPositionY = value;
                OnPropertyChanged();
            }
        }
```
